Sources/ServiceKit/Container/Container.swift

```diff
@@ -52,6 +52,8 @@ extension Container {
     /// - throws: Any error finding or initializing the requested service.
     /// - returns: Initialized instance of `T`
     public func make<S>(_ service: S.Type = S.self) throws -> S {
+        assert(!self.didShutdown, "Container.shutdown() has been called, this Container is no longer valid.")
```

Comment:
(see above comment)
This would than become `assert(self.isActive, "...")` which IMO reads nicer